src/client/app/components/kid-view/KidReminder.jsx

```diff
@@ -4,41 +4,38 @@ import React, { Component } from 'react';
 
 class KidReminder extends Component {
   constructor(props) {
-        super(props);
-		this.state = { reminder: '', reminders:[] };
-
-		this.handleChange = this.handleChange.bind(this);
+    super(props);
+    this.state = { reminder: '', reminders:[], placeholder:'Type new reminder' };
+		
+    this.handleChange = this.handleChange.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
     this.resetReminders = this.resetReminders.bind(this);
 	}
 
-	handleChange(event) {
-		this.setState( {reminder: event.target.value } );
-	}
-
-	onSubmit() {
-		event.preventDefault();
-		var existingReminders;
-		if (this.state.reminders == ''){
-			existingReminders = [];
-		} else {
-			existingReminders = this.state.reminders;
-		}
-		existingReminders.push(this.state.reminder);
-		this.setState({reminders: existingReminders});
-	}
+  handleChange(event) {
+    this.setState( {reminder: event.target.value } );
+  }
 
-	illustrateReminders(){
-		if (this.state.reminders != ''){
-			return this.state.reminders.map(reminder =>
-				<li>{reminder}</li>
-			);
-		}
+  onSubmit() {
+    if (this.state.reminder !== ''){
+      event.preventDefault();
+      var existingReminders =  this.state.reminders;
+      existingReminders.push(this.state.reminder);
+      this.setState({reminders: existingReminders, reminder: '', placeholder: 'Type new reminder'});
+    } else {
+      this.setState({placeholder:'Reminder is required to submit'});
+    }  
   }
 
+  illustrateReminders(){
+    return this.state.reminders.map((reminder,index) =>
+      <li key={index}>{reminder}</li>
+    );
+  }
+  
   resetReminders(){
     event.preventDefault();
```

Comment:
Needed? 
I'm leaning towards no. This preventDefault is used to prevent the reload of a page by a forms submit action but this is not a form, just a div with a button. 
Does it work without this?